Sprint-1/interpret/to-pounds.js

```diff
@@ -1,19 +1,21 @@
-const penceString = "399p";
-
-const penceStringWithoutTrailingP = penceString.substring(
+// variable declaration; assignment of value 399p to penceString variable
+const penceString = "399p"; 
+// variable declaration; the expression takes the p out of penceString and outputs 399
+const penceStringWithoutTrailingP = penceString.substring( 
   0,
   penceString.length - 1
 );
-
+//variable declaration; the expression adds 0 to the start of the penceStringWithoutTrailingP if it has less than 3 characters
 const paddedPenceNumberString = penceStringWithoutTrailingP.padStart(3, "0");
+//variable declaration; the expression takes out 99 from paddedPenceNumberString and outputs 3
 const pounds = paddedPenceNumberString.substring(
   0,
   paddedPenceNumberString.length - 2
 );
-
+//variable declaration; the expression takes out 3 from paddedPenceNumberString, then adds 0 to the end of if its characters are less than 2
```

Comment:
- Can you explain as if `penceString` can have any valid value instead of just "399p"? That is, for general cases. 
  - Otherwise if I simply read this line of comment, I would wonder "what is this '3' in the comment referring to".

- Do you think this code will still work correctly if we remove `.padEnd(2, "0")`?